app/assets/stylesheets/organizations.scss

```diff
@@ -16,6 +16,9 @@
   }
 }
 
+// Access bootstrap color utility variables
+@import 'bootstrap';
```

Comment:
Is this needed? We're already importing bootstrap in application.scss.

Reply:
I had undefined variables unless I did this - not sure about the deeper implications for asset loading.